### PR TITLE
Interceptors assignment

### PR DESCRIPTION
**Pull request by AkashBoora**
Please review the code

### CODE REVIEW

File: authentication/client/src/components/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+interface Props {
+  setToken: (arg: string) => void;
+}
+
+export default function Login(props: Props) {
+  const [name, setName] = useState<String>("");
+  const [password, setPassword] = useState<String>("");
+
+  const [getlogin, { error, data }] = useLazyQuery(GET_LOGIN, {
```
Comment (gokul-krshna) [inline authentication/client/src/components/Login.tsx]: lets use loading state as well, the state sequence should be `loading ? spinner : (error ? errorMessage ? data)`

File: twitter-feed-query/resolvers/Mutation.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  deleteTweet: (parent, { id }, { db }) => {
+    const deletedTweet = db.tweets.find((tweet) => {
+      if (tweet.id === id) {
+        tweet.authorId = null;
+        tweet.statsId = null;
+        return true;
+      }
+    });
+    const idInt = id.slice(5);
+    console.log(idInt);
```
Comment (gokul-krshna) [inline twitter-feed-query/resolvers/Mutation.js]: remove logs

File: interceptor-logging/src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+// axios.interceptors.request.use((request) => {
+//   console.log("API call is requested");
+//   console.log(request);
+//   return request;
+// });
+
+
+// axios.interceptors.response.use((response) => {
+//   if (response.status === 200) {
```
Comment (gokul-krshna) [inline interceptor-logging/src/index.tsx]: remove comments

File: authentication/client/src/reportWebVitals.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { ReportHandler } from 'web-vitals';
+
+const reportWebVitals = (onPerfEntry?: ReportHandler) => {
```
Comment (gokul-krshna) [inline authentication/client/src/reportWebVitals.ts]: remove this file, if not required

**Comment by AkashBoora** [comment]
Please review now gokul!

**Pull request merged**